tests/test_power_spectrum_and_bfactor.py

```diff
@@ -0,0 +1,72 @@
+import torch
+from cryo_challenge.power_spectrum_utils import _centered_ifftn, compute_power_spectrum
+from cryo_challenge._preprocessing.bfactor_normalize import (
+    _compute_bfactor_scaling,
+    bfactor_normalize_volumes,
+)
+
+
+def test_compute_power_spectrum():
+    box_size = 224
+    volume_shape = (box_size, box_size, box_size)
+    voxel_size = 1.073 * 2
+
+    freq = torch.fft.fftshift(torch.fft.fftfreq(box_size, d=voxel_size))
+    x = freq.clone()
+    y = freq.clone()
+    z = freq.clone()
+    x, y, z = torch.meshgrid(x, y, z, indexing="ij")
+
+    s2 = x**2 + y**2 + z**2
+
+    b_factor = 170
+
+    gaussian_volume = torch.exp(-b_factor / 4 * s2).reshape(volume_shape)
+    gaussian_volume = _centered_ifftn(gaussian_volume)
+
+    power_spectrum = compute_power_spectrum(gaussian_volume)
+    power_spectrum_slice = (
+        torch.abs(torch.fft.fftn(gaussian_volume)[: box_size // 2, 0, 0]) ** 2
+    )
+
+    mean_squared_error = torch.mean((power_spectrum - power_spectrum_slice) ** 2)
+
+    assert mean_squared_error < 1e-3
+
+    return
+
+
+def test_bfactor_normalize_volumes():
```

Comment:
I'd put a docstring about how this test works. 

Does the volume need to be oscillatory, or was that just one example? Could it be random?

Reply:
A random volume would not be a good test because we would not know what to expect. It would be good to check that everything runs, but this test also takes care of that.